File: september/25-09/25-09.py
def second_largest(arr):

    #sort array
    arr.sort()
    sort_list = []

    for element in arr:
        if(element not in sort_list):
            sort_list.append(element)

    second = sort_list[len(sort_list)-2]
    print(second)

    return second

#Tests:
second_largest([1, 2, 3, 4]) #should return 3.
second_largest([20, 139, 94, 67, 31]) #should return 94.
second_largest([2, 3, 4, 6, 6]) #should return 4.
second_largest([10, -17, 55.5, 44, 91, 0]) #should return 55.5.
second_largest([1, 0, -1, 0, 1, 0, -1, 1, 0]) #should return 0.

File: september/25-09/25-09.js
function secondLargest(arr) {

    arr.sort((a,b) => a -b );
    
    let unique_list = [];

    for (const element of arr) {

        if(!unique_list.includes(element)){
            unique_list.push(element);
        }
        
    }

    let second = unique_list[unique_list.length -2];
    console.log(second);

    return second;
}